mycroft/messagebus/client/ws.py

```diff
@@ -41,6 +41,10 @@ def __init__(self, host=client_config.get("host"),
                  port=client_config.get("port"),
                  path=client_config.get("route"),
                  ssl=str2bool(client_config.get("ssl"))):
+        if not host:
+            # By default open localhost if host is empty, or None
+            host = "127.0.0.1"
```

Comment:
Actually, if `host` or any other variable from the config does not exist, we'll face an exception. In order to prevent that we should have a default parameter during the `.get` operation, for instance: 
```
...
host=client_config.get("host", "localhost"),
port=client_config.get("port", 8080),
...
```